CONTRIBUTING.md

```diff
@@ -1,91 +1,101 @@
 # Contributing to Bisq
 
-Anyone is welcome to contribute to Bisq. This document provides an overview of how we work. If you're looking for somewhere to start contributing, see the [good first issue](https://github.com/bisq-network/bisq/issues?q=is%3Aopen+is%3Aissue+label%3A"good+first+issue") list.
+Read on to find out if taking part in the development of Bisq is for you and how to do it!
 
+## Are you "the guy"?
 
-## Communication Channels
+Java God, Bitcoin Native, Marketing Genius, Cypherpunk, Testing Machine, or simply someone who likes the idea of Bisq? Anyone is welcome to contribute! However, Bisq is no classic organization, there is no boss, there is no working hours and there is no classic payment. In short, Bisq is not for everyone.
 
-Most communication about Bisq happens in the [Bisq Slack workspace](https://bisq.network/slack-invite).
+**What is there to work on?** You have to be able to find and work a problem on your own. In general, we do not assign stuff to people, there is no specification to be implemented, there is no given way forward. You yourself have to find, choose and pick something to work on. If you get stuck, we are keen to help, but please do not wait for us to take your hand. Also, you can spend as much (or little) time as you want and when you want to - Bisq still is an open source project.
 
-Discussion about code changes happens in GitHub issues and pull requests.
+**Will you get paid?** Well, yes and no. With Bisq, there is no classic funding or investors, so there is no payout of fiat money.
+- Yes, because there is the DAO. The DAO allows for compensation requests to be filed and voted on. If such a voting goes in your favor, you receive BSQ, a colored bitcoin token. Sell your BSQ and you get your fiat eventually.
+- No, because the market is still very small. We can sustain a few full-timers (paying rent, tax, small stuff) but do not expect to get rich fast and easy just now.
```

Comment:
agreed, we should not make such assumptions. However, 0,xx BTC per day is hard to grasp. How about:

`- No, because the BSQ market is still bootstrapping. For the first months, we saw BSQ sold for a total of 8 BTC per month vs. about 20 compensation requests. So there is some payment happening but do not expect to get rich fast and easy just now.`

The numbers of course have to be properly fetched, I just eyeballed them. And, downside is that we might have to correct the numbers once in a while (I fear that this is not going to happen).

How about something like
`- No, because the BSQ market is still bootstrapping. Although BSQ already has a healthy sell rate, it is not yet enough to sustain all compensation requests. (So do not expect to get rich fast and easy just now.)`

Reply:
I think we should not "freeze" the currently bad market (caused by bad trade volume) in a doc like that. The first 4 months have been liquid enough to sell BSQ, just over the past 3-4 months trade volume and with that burned BSQ broke down for various reasons (Bitcoin bear market, our Monereo whales seems to have left, 1.2 update and the related problems and bugs,...). A neutral market overview would be better IMO or simply point to the markets page so people can draw their own picture.